libr/include/r_util/r_strbuf.h

```diff
@@ -19,12 +19,14 @@ R_API bool r_strbuf_set(RStrBuf *sb, const char *s);
 R_API bool r_strbuf_slice(RStrBuf *sb, int from, int len);
 R_API bool r_strbuf_setbin(RStrBuf *sb, const ut8 *s, size_t len);
 R_API ut8* r_strbuf_getbin(RStrBuf *sb, int *len);
-R_API bool r_strbuf_setf(RStrBuf *sb, const char *fmt, ...);
+R_API bool r_strbuf_setf(RStrBuf *sb, const char *fmt, ...)
```

Comment:
I know it becomes a bit longer, but I think putting it on the same line would be better. @trufae opinions?

Reply:
Fixed as well, it will be in a separate PR because it generates many warnings.